libraries/ae-inventory-processor/src/main/java/org/metaeffekt/core/inventory/processor/model/Constants.java

```diff
@@ -15,15 +15,16 @@
  */
 package org.metaeffekt.core.inventory.processor.model;
 
-import org.metaeffekt.core.util.FileUtils;
```

Comment:
@karsten-klein is this neccessary for any other modules or is it fine if I throw it out, since it was unused :)